frontend/assets/scss/colors.scss

```diff
@@ -19,7 +19,6 @@
   --color-red-alert: #d51548;
   --color-accent-magenta: #f7147d;
   --color-error-background: #fae6e9;
-  --color-red-alert: #bf1a2f;
```

Comment:
This is a duplicate, remove this one because it didn't match the colors from platform.